apps/prairielearn/elements/pl-code/pl-code.mustache

```diff
@@ -10,9 +10,11 @@ $(function(){
     style="--pl-code-background-color: {{background_color}}; --pl-code-line-number-color: {{line_number_color}};">
     <div
         id="pl-code-{{uuid}}"
+        {{^prevent_select}}
         class="rounded"
+        {{/prevent_select}}
         {{#prevent_select}}
-        style="-moz-user-select: none; -webkit-user-select: none; -ms-user-select:none; user-select:none;-o-user-select:none;"
+        class="rounded pl-code-prevent-select"
```

Comment:
Bootstrap has a `user-select-none` class; should we be using that instead?

https://getbootstrap.com/docs/5.3/utilities/interactions/#text-selection

Reply:
![image](https://github.com/user-attachments/assets/843d04d9-106d-4f02-b508-c025f67dd029)
Seems to work. [4881bb4](https://github.com/PrairieLearn/PrairieLearn/pull/11939/commits/4881bb432b5076da7d305897949b1e41bb4a0657)